addons/mail/static/src/composer/composer.js

```diff
@@ -472,7 +472,9 @@ export class Composer extends Component {
                 attachments: this.props.composer.attachments,
                 isNote:
                     this.props.composer.type === "note" ||
-                    this.props.messageToReplyTo?.message?.isNote,
+                    this.props.messageToReplyTo?.message?.isNote ||
+                    (this.env.inChatWindow &&
+                        this.props.composer.thread.model !== "discuss.channel"),
```

Comment:
All those should be in composer type, not hacked at message post. So that state is correct, in particular "log note..." / "message ..." placeholder